### PR TITLE
Created `package.json` files

### LINKED ISSUE #33: Add package.json to make installation easier
I've been getting a lot of emails wondering how to install dependencies. I'd like to point them to a `package.json` for easy installation.

### PR DESCRIPTION
**Pull request by xmwx38**
Fixes #33 

- Created `package.json` file for root chapters
- Added `steam-totp` to Chapter 1.2 - Prerequisites/`README.md`

### CODE REVIEW

File: Chapter 1 - Basics/package.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "project1",
+  "version": "1.0.1",
+  "description": "JS Steam bots project, built following ~ https://github.com/andrewda/node-steam-guide",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Default User",
+  "license": "ISC",
```
Comment (andrewda) [inline Chapter 1 - Basics/package.json]: License is Creative Commons (see the README)

Comment (andrewda) [inline Chapter 1 - Basics/package.json]: I think you can use `CC-BY-4.0` for the license identifier.

Comment (xmwx38) [inline Chapter 1 - Basics/package.json]: @andrewda I've edited the license to `CC-BY-4.0` & the author to "Default."

File: Chapter 1 - Basics/package.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "project1",
+  "version": "1.0.1",
+  "description": "JS Steam bots project, built following ~ https://github.com/andrewda/node-steam-guide",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Default",
```
Comment (andrewda) [inline Chapter 1 - Basics/package.json]: I don't think `author` is a required field. Can we just remove it?

File: Chapter 1 - Basics/package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "project1",
+  "version": "1.0.1",
+  "description": "JS Steam bots project, built following ~ https://github.com/andrewda/node-steam-guide",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (andrewda) [inline Chapter 1 - Basics/package.json]: I'm not going to test anything. Let's get rid of these scripts.

File: Chapter 1 - Basics/package.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "project1",
+  "version": "1.0.1",
+  "description": "JS Steam bots project, built following ~ https://github.com/andrewda/node-steam-guide",
+  "main": "index.js",
```
Comment (andrewda) [inline Chapter 1 - Basics/package.json]: There's no `index.js` file in this directory, so this fake `main` file should be removed.

File: Chapter 1 - Basics/package.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "project1",
+  "version": "1.0.1",
```
Comment (andrewda) [inline Chapter 1 - Basics/package.json]: Can we keep the version as `1.0.0` for all of these? The thing that changes is the name (`project1`, `project2`, etc.) not the version.

File: Chapter 1 - Basics/package.json [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+  "description": "JS Steam bots project, built following ~ https://github.com/andrewda/node-steam-guide",
```
Comment (andrewda) [inline Chapter 1 - Basics/package.json]: To remove as much clutter as possible, let's get rid of this too. Descriptions are mainly for npm packages, but people know where they are here.

**Comment by xmwx38** [comment]
The `package.json` files now look like the following example.

   `"name": "project1",`
   `"version": "1.0.0",`  
    `"license": "CC-BY-4.0",`  
    `"dependencies": {`

File: Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js [hunk 7]
```diff
@@ -65,7 +65,7 @@ passport.use(new SteamStrategy({
 ));
 
 io.use(passportSocket.authorize({
-	cookieParser: cookieParser,
+	cookieParser: cookieParser
```
Comment (andrewda) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: This is gunna cause a crash

Comment (xmwx38) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: @andrewda I based the edits on the Codacy code style issues https://www.codacy.com/app/andrewda/node-steam-guide/dashboard

Comment (xmwx38) [inline Chapter 6 - Connecting Sites and Bots/Chapter 6.3 - Beginning the Connection/project10/app.js]: I reversed the edits to `app.js`

**Pull request merged**